recipe/meta.yaml

```diff
@@ -76,6 +82,22 @@ test:
     - test -f ${PREFIX}/include/{{ each_ncurses_header }}
     {% endfor %}
 
+    # Test pkg-config files
+    {% set ncurses_pc_files = [
+        "form.pc",
+        "menu.pc",
+        "ncurses++.pc",
+        "ncurses.pc",
+        "panel.pc",
+        "tinfo.pc"
+    ] %}
+    {% for each_ncurses_pc_file in ncurses_pc_files %}
+    - test -f ${PREFIX}/lib/pkgconfig/{{ each_ncurses_pc_file }}
+    {% endfor %}
+
+    # Test ncurses library arguments.
+    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo $"
```

Comment:
Verifies that `libncurses` and `libtinfo` will both be linked against when using `pkg-config`.